ucp/useraccounts_module.php

```diff
@@ -319,143 +269,146 @@ function main($id, $mode)
 						trigger_error($this->language->lang('FORM_INVALID') . '<br><br>' . $this->user->lang('RETURN_UCP', '<a href="' . $this->u_action . '">', '</a>'));
 					}
 
-					$delete_sql = empty($markedbadges) ? '' : ' AND ' . $this->db->sql_in_set('badge_id', $markedbadges, true); // true for NOT IN, false (default) for IN
-
 					// Delete all user badges that are NOT IN the array of marked badges or all if none is selected.
+					$delete_sql = empty($markedbadges) ? '' : ' AND ' . $this->db->sql_in_set('badge_id', $markedbadges, true); // true for NOT IN, false (default) for IN
 					$sql = 'DELETE FROM ' . $user_badge_table . '
 						WHERE user_id = ' . $this_user_id . "
```

Comment:
Switching between `'`' and `"` style quotes within the same statement isn't great.  Can you please stick with `'` and concatenating in variables? This seems to be the preferred convention in most of the phpbb code.

Reply:
This is  phpBB modifies queries. Generally, `"` is more comfortable because it allows to write variables in the sql statement without any `'  . $var . '`, however `$this->user` does not work in when written in double quotes. 

I would like to keep the style for the cases when we modify queries because I'm not sure how the alternative writing would be and this is default phpBB.

Reply:
Why are you using `$this_user_id` with single quotes and concatenation?